examples/rfmip-clear-sky/rrtmgp_rfmip_sw.F90

```diff
@@ -265,25 +267,27 @@ program rrtmgp_rfmip_sw
 #endif
     ! Boundary conditions
     !   (This is partly to show how to keep work on GPUs using OpenACC in a host application)
-    !
     ! What's the total solar irradiance assumed by RRTMGP?
-    !  The first two loops could be more expressed more ompactly as def_tsi(1:block_size) = sum(toa_flux, dim=2)
     !
-    !!!$acc parallel loop
-    do icol = 1, block_size
-      def_tsi(icol) = toa_flux(icol, 1)
-    end do
-    !!!$acc parallel loop collapse(2)
+#ifdef _OPENACC
+    call zero_array(block_size, def_tsi)
+    !$acc parallel loop collapse(2) copyout(def_tsi) copyin(toa_flux)
```

Comment:
def_tsi is aggregated on the GPU, but initialized on the CPU. Should this have a copyin (or just copy) here instead of just a copyout?

Reply:
Hey, this was a good suggestion, thanks. What's a bummer is that creating the toa_flux array on the GPUs leads to some missing values. You can see by uncommenting the acc create/delete pair for this array. Wish I understood why...